sorc/chgres_cube.fd/search_util.f90

```diff
@@ -198,6 +202,10 @@ subroutine search (field, mask, idim, jdim, tile, field_num, latitude, terrain_l
 
  end subroutine search
 
+!> set sst values based on latitude
+!!
+!! @param latitude      - latitude input
+!! @param sst           - sst guess value to be set
```

Comment:
Author tag please.

Reply:
brief tag added

Reply:
We do not need brief tag on subroutines/functions. We do need author tag.

Reply:
Removed brief and add author tag